packages/cli/src/amplify.ts

```diff
@@ -43,5 +43,8 @@ try {
 
   await usageDataEmitter.emitSuccess({}, metricDimension);
 } catch (e) {
-  printer.log('Failed to emit usage metrics', LogLevel.DEBUG);
+  if (e instanceof Error) {
+    printer.log(format.error('Failed to emit usage metrics'), LogLevel.DEBUG);
+    printer.log(format.error(e), LogLevel.DEBUG);
```

Comment:
I think it's okay for the formatter to not include the stack trace, but we should have another debug log for it here

Reply:
or `format.error(e, { includeStack: true })`

Reply:
I think @sobolk's approach would return long ugly string. 

I think the output will look cleaner just by add extra log statement
`if (e.stack) { printer.log(format.error(e.stack)) }`